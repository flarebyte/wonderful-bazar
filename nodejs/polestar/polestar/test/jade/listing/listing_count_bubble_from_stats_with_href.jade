include mixins/listing
ul
  mixin listing_count_bubble_from_stats_with_href(everything.title, everything.item,everything.href, everything.statKey)
  mixin listing_count_bubble_from_stats_with_href(no_item.title, no_item.item, no_item.href, no_item.statKey)
  mixin listing_count_bubble_from_stats_with_href(item_is_null.title, item_is_null.item, item_is_null.href, item_is_null.statKey)
  mixin listing_count_bubble_from_stats_with_href(stats_is_null.title, stats_is_null.item, stats_is_null.href, stats_is_null.statKey)
  mixin listing_count_bubble_from_stats_with_href(no_statKey_value.title, no_statKey_value.item, no_statKey_value.href, no_statKey_value.statKey)
