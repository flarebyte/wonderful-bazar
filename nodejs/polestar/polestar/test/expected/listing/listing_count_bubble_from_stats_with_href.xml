<ul><li><a href="#there">everything<span class="ui-li-count">12</span></a></li><li><a href="#there">no_statKey_value</a></li></ul>